commands/preprocess.py

```diff
@@ -173,6 +173,28 @@ def transform_loader_link(target, file, root):
     abstarget = os.path.join(root, "common/" + convert_loader_name(target))
     return os.path.relpath(abstarget, os.path.dirname(file))
 
+def is_ranges_placeholder(target):
```

Comment:
Can we have a unit test for this function? `tests/test_preprocessor.py` might be a good place.